### ISSUE TITLE
v-bind shorthand don’t work with 9.20

### ISSUE THREAD

**Issue by matteo6b** [vuejs/eslint-plugin-vue#2365]
**Checklist**

- [x ] I have tried restarting my IDE and the issue persists.
- [x ] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:**  8.56.0
- **eslint-plugin-vue version:** 9.20.0
- **Node version:**  20.9.0
- **Operating System:** windows

**Please show your full configuration:**

```require('@rushstack/eslint-patch/modern-module-resolution')

module.exports = {
  root: true,
  plugins: ['vuejs-accessibility'],
  extends: [
    'plugin:vue/vue3-recommended',
    'eslint:recommended',
    '@vue/eslint-config-typescript',
    '@vue/eslint-config-prettier/skip-formatting',
    'plugin:vuejs-accessibility/recommended'
  ],
   parser: "vue-eslint-parser",
  parserOptions: {
    ecmaVersion: 'latest'
  }
}
```

**What did you do?**	

```
 <Modal  :id="id" v-model="showModal"  :id-screen /> 
```

**What did you expect to happen?**
No errors with v-bind shorthand

**What actually happened?**
v-bind is missing expression.vue(34)

**Comment by FloEdelmann** [comment]
Which Vue.js version are you using?

**Comment by matteo6b** [comment]
"vue": "3.4.7",

**Comment by FloEdelmann** [comment]
>  v-bind is missing expression.vue(34)

This message was not reported by ESLint  or eslint-plugin-vue. Maybe it's only an editor warning? Did you try restarting your IDE?

**Comment by matteo6b** [comment]
> > v-bind is missing expression.vue(34)
> 
> This message was not reported by ESLint or eslint-plugin-vue. Maybe it's only an editor warning? Did you try restarting your IDE?

yes

**Comment by FloEdelmann** [comment]
Then please provide a reproduction repository or Stackblitz, so we can try to debug the issue.

**Comment by matteo6b** [comment]
https://github.com/matteo6b/test-elint

**Comment by matteo6b** [comment]
I believe that message is coming from Vue Language Tools rather than ESLint. See https://github.com/vuejs/language-tools/issues/3830.

**Comment by FloEdelmann** [comment]
Yes that makes sense. Thank for linking that issue. I'll close this one then.
